Public/Src/Utilities/Native/IO/Windows/FileUtilities.Win.cs

```diff
@@ -279,7 +279,7 @@ public static string NormalizeDirectoryPath(string path)
                             // Deletion is successful when the child directories/files count matches the expected count
                             return actualRemainingChildCount == remainingChildCount;
                         },
-                        rethrowExceptions: true /* exceptions thrown in work() will occur again on retry, so just rethrow them */);
+                        onException: e => throw e /* exceptions thrown in work() will occur again on retry, so just rethrow them */);

```

Comment:
Just sharing that this will change the callstack of the error thrown to be shown here, rather than in the original callstack since we used to use `throw`

Reply:
Good catch! Yeah, I should've tried to preserved the callstack.